benchmark.py

```diff
@@ -20,12 +23,10 @@
 # default configuration options
 conf = {
     "working_dir": benchmark_dir,
+    "repo_dir":    "repos",
```

Comment:
Am I reading this wrong, or is this bringing back in stuff I was getting rid of in my most recent merge?